### PR TITLE
frontend/feat: Added Referral Stats View

### PR DESCRIPTION
**Pull request by PraveenGongada**
## Type of Change

- [ ] Bugfix
- [x] New feature
- [ ] Enhancement
- [ ] Refactoring
- [ ] Dependency updates

## Description

Added Referral Stats View

### CODE REVIEW

File: Frontend/ui-driver/src/Screens/Benefits/BenefitsScreen/View.purs [hunk 1]
```diff
@@ -354,8 +499,8 @@ driverReferralCode push state =
        , cornerRadius 8.0
        , background Color.lightBlue80
        , gravity CENTER_VERTICAL
-       , visibility $ boolToVisibility $ state.props.isPayoutEnabled == Just true && state.props.driverReferralType == CUSTOMER
-       , onClick push $ const $ GoToCustomerReferralTracker
+       , visibility $ GONE
```
Comment (MercyQueen) [inline Frontend/ui-driver/src/Screens/Benefits/BenefitsScreen/View.purs:502]: @PraveenGongada  why is the visibility set to GONE?

File: Frontend/ui-driver/src/RemoteConfigs/Utils.purs [hunk 2]
```diff
@@ -125,4 +125,10 @@ getReferralPopUpDelays popUpType = do
       ST.ReferNow -> value.refer_now
       ST.AddUPI -> value.add_upi
       ST.VerifyUPI -> value.verify_upi
-      _ -> oneDayInMS
\ No newline at end of file
+      _ -> oneDayInMS
+
+getReferralBonusVideo :: String -> String 
+getReferralBonusVideo city = 
+  let config = fetchRemoteConfigString "referral_bonus_videos"
```
Comment (MercyQueen) [inline Frontend/ui-driver/src/RemoteConfigs/Utils.purs:132]: is this added to remote config?

Comment (PraveenGongada) [reply Frontend/ui-driver/src/RemoteConfigs/Utils.purs:132]: no

File: Frontend/ui-driver/src/Screens/Benefits/BenefitsScreen/View.purs (hunk 1 above)
Comment (MercyQueen) [reply Frontend/ui-driver/src/Screens/Benefits/BenefitsScreen/View.purs:502]: sure.

**Review by MercyQueen — APPROVED** [review]

**Pull request merged**